test/controllers/user.js

```diff
@@ -194,5 +198,144 @@ describe('/POST users/sessions', () => {
 
 });
 
+/*
+* Testing the /users/list (POST) route
+*/
+describe('/POST users/list', () => {
+
+  const userOne = {
+    name: 'userOne',
+    lastName: 'userOne',
+    email: 'userOne@wolox.com',
+    password: '12345678'
+  };
+
+  const userTwo = {
+    name: 'userTwo',
+    lastName: 'userTwo',
+    email: 'userTwo@wolox.com',
+    password: '12345678'
+  };
+
+  it('should return all users from the database', (done) => {
+
+    chai.request(server)
+      .post('/users')
+      .send(newUser)
+      .then((res) => {
+
+        chai.request(server)
+          .post('/users')
+          .send(userOne)
+          .then((res) => {
+
+            chai.request(server)
+              .post('/users')
+              .send(userTwo)
+              .then((res) => {
+
+                chai.request(server)
+                  .get('/users/list')
+                  .set('token', token)
+                  .then((res) => {
+                    res.should.have.status(200);
+                    res.body.should.include(
+                      {name: 'Kevin', lastName: 'Temes', email: 'kevin.temes@wolox.com.ar'},
+                      {name: 'userOne', lastName: 'userOne', email: 'userOne@wolox.com'}, 
+                      {name: 'userTwo', lastName: 'userTwo', email: 'userTwo@wolox.com'}
+                    );
+                    dictum.chai(res, 'User list retrieval');
+                    done();
+                  });
+                
+                
+              });
+            
+          });
+        
+      });
+
+  });
+
+  it('should return only the second user', (done) => {
+
+    chai.request(server)
```

Comment:
To avoid repeting this request you can move it to a beforeEach or a before in this scope.